### PR TITLE
Bug 1982659: Workloads - Jobs  'Type' column's Value 'Non-parallel' is in English 

### PR DESCRIPTION
**Pull request by kdoberst**
Addresses [Bug 1982659](https://bugzilla.redhat.com/show_bug.cgi?id=1982659)

In the Jobs table the values listed in the "Type" column are i18n ready.

### CODE REVIEW

File: frontend/public/locales/en/public.json [hunk 1]
```diff
@@ -1682,6 +1682,9 @@
   "ConsolePlugins": "ConsolePlugins",
   "Open Support Case with Red Hat": "Open Support Case with Red Hat",
   "Report Bug to Red Hat": "Report Bug to Red Hat",
+  "Non-parallel": "Non-parallel",
+  "Fixed Completion Count": "Fixed Completion Count",
```
Comment (zherman0) [inline frontend/public/locales/en/public.json:1686]: Should we change these to follow the new capitalization rules: "Fixed completion count?"

Comment (kdoberst) [reply frontend/public/locales/en/public.json:1686]: Good point. So would it also be "Work queue" instead of "Work Queue"?

Comment (spadgett) [reply frontend/public/locales/en/public.json:1686]: Yes, if these are table columns, they should use sentence case.

Comment (kdoberst) [reply frontend/public/locales/en/public.json]: Changed the capitalization rules.

**Review by spadgett — APPROVED** [review]
/lgtm

**Review by zherman0 — APPROVED** [review]

**Pull request merged**